tests/api_validation_test.go

```diff
@@ -322,7 +323,7 @@ var _ = Describe("[rfe_id:1130][crit:medium][posneg:negative][vendor:cnv-qe@redh
 			By(fmt.Sprintf("Waiting for datavolume to match phase %s", string(cdiv1.ImportInProgress)))
 			err = utils.WaitForDataVolumePhase(f, f.Namespace.Name, cdiv1.ImportInProgress, dataVolumeName)
 			if err != nil {
-				f.PrintControllerLog()
+				klog.Warningf("failed to wait for DataVolume phase: %v", err)
```

Comment:
Not the worst thing ever but pretty suprised to see klog in the tests package, usually have to use the ginkgo writer https://github.com/kubevirt/containerized-data-importer/blob/7c93c54bac55dc4becdad354ab9be7c1fa8bf34a/tests/api_validation_test.go#L269

Reply:
I'll replace it.